paper-ripple-behavior.html

```diff
@@ -66,10 +66,10 @@
     /**
      * Ensures this element contains a ripple effect. For startup efficiency
      * the ripple effect is dynamically on demand when needed.
-     * @param {!Event=} opt_triggeringEvent (optional) event that triggered the
+     * @param {!Event=} optTriggeringEvent (optional) event that triggered the
      * ripple.
      */
-    ensureRipple: function(opt_triggeringEvent) {
+    ensureRipple: function(optTriggeringEvent) {
```

Comment:
I think that I would ask people to use `opt_` while acting as a readability reviewer and that this style is "preferred" by the style guide (and used consistently within it) and is incredibly prevalent within Google and Closure (and Chrome and ...).

what polymer really needs to figure out is whether this precedence matters to it :).